brouter-expressions/src/main/java/btools/expressions/BExpressionContext.java

```diff
@@ -565,85 +565,85 @@ public BExpressionLookupValue addLookupValue(String name, String value, int[] lo
           String org = value;
           try {
             // remove some unused characters
-            value = value.replace(",", ".");
-            value = value.replace(">", "");
-            value = value.replace("_", "");
+            value = value.replaceAll(",", ".");
+            value = value.replaceAll(">", "");
+            value = value.replaceAll("_", "");
+            value = value.replaceAll(" ", "");
+            //value = value.replaceAll("�", ""); // nbsp
+            value = value.replaceAll("�", "'");
+            value = value.replaceAll("�", "\"");
             if (value.indexOf("-") == 0) value = value.substring(1);
-            if (value.indexOf("~") == 0) value = value.substring(1);
-            if (value.contains("-")) {    // replace eg. 1.4-1.6 m
-              String tmp = value.substring(value.indexOf("-") + 1).replaceAll("[0-9.,-]", "");
-              value = value.substring(0, value.indexOf("-")) + tmp;
+            if (value.contains("-")) {    // replace eg. 1.4-1.6 m but also 1'-6"
+              value = value.substring(0, value.indexOf("-")); // + tmp;
```

Comment:
Won't this fail on something like `30-35 mph` because it drops the unit which is needed for the later conversion?

Reply:
Yes, in the old version `2m-3m` returns as 2mm. So I drop this.
But anyway this is not a good solution for all. I have a version with numeric check before the append.

Reply:
>  errors in the build log

Interesting thing. No, I haven't seen. The build was 'green' so I didn't look inside the action report. 
On Windows I didn't get the errors and the rd5 generation error log shows the expected results.
But I checked the lib included in ZIP file (from last action). This is not usable - in this case.

Reply:
> `2m-3m` returns as 2mm

Good catch! I had not thought of this case at all.